pkg/cmd/util/clientcmd/factory_object_mapping.go

```diff
@@ -61,6 +62,37 @@ func (f *ring1Factory) UnstructuredObject() (meta.RESTMapper, runtime.ObjectType
 	return f.kubeObjectMappingFactory.UnstructuredObject()
 }
 
+func (f *ring1Factory) CategoryExpander() resource.CategoryExpander {
```

Comment:
For now, (we have yet to bring in the new `factory_builder.go#NewBuilder`), it is used [in place of `resource.NewBuilder`](https://github.com/openshift/origin/pull/15100/files#diff-544b29a7b499f22b6e316a89c6e84f36)